src/post_and_poll.py

```diff
@@ -67,6 +67,6 @@ def get_run_uuid(post, API_KEY, api_url):
     run_uuid = "my_run_id"
     file_name = "my_results.json"
     api_url = 'https://developer.nrel.gov/api/reopt/v1'
-    from keys import API_KEY
+    API_KEY = "DEMO_KEY"
```

Comment:
@NLaws do we want DEMO_KEY in here? That's a valid key that will return responses. Should we load it from the environment?

Reply:
yes this is to help people get started. we could use the keys.py but I felt like that added another layer of abstraction. Just trying to keep it as simple as possible.